src/plots/plots.js

```diff
@@ -467,6 +466,7 @@ plots.supplyDefaults = function(gd, opts) {
     newFullLayout._hasGL2D = newFullLayout._has('gl2d');
     newFullLayout._hasTernary = newFullLayout._has('ternary');
     newFullLayout._hasPie = newFullLayout._has('pie');
+    newFullLayout._hasFunnelarea = newFullLayout._has('funnelarea');
```

Comment:
No need for this. Please :hocho: 

These `fullLayout._has??` fields are just there for backward compatibility (mostly for our old toolpanel code). We shouldn't add new ones. If you need to add a `funnelarea`-only block somewhere downstream, you should use `fullLayout._has('funnelarea')` instead.

Reply:
Done in https://github.com/plotly/plotly.js/pull/3876/commits/abc3b39d159e5d4f53dc170b1d11f4fc3bc5f535.